apps/acceptance/clock/acceptanceStopwatch.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+var Clock = require('./regions/appView');
```

Comment:
File names use usually underscores as separator. It helps to prevent conflicts when somebody clone the project on a case-insensitive file system.